sleap_io/io/nwb.py

```diff
@@ -198,9 +423,13 @@ def write_nwb(
     )
 
     nwbfile = NWBFile(**nwb_file_kwargs)
-    nwbfile = append_nwb_data(labels, nwbfile, pose_estimation_metadata)
+    if as_training:
+        nwbfile = append_nwb_training(labels, nwbfile, pose_estimation_metadata)
+    else:
+        nwbfile = append_nwb_data(labels, nwbfile, pose_estimation_metadata)
 
     with NWBHDF5IO(str(nwbfile_path), "w") as io:
+        #io.nwb_version = (2.0, "2.0.0") #TODO figure out how to set version
```

Comment:
The Python file is called `nwb.py`, so that results in a circular import. I have to set the version of the `.nwb` file that is created, not the version of the entire repository.